config/core/test-configs.yaml

```diff
@@ -1640,6 +1667,19 @@ device_types:
             - 'tinyconfig'
             - 'kvm_guest.config'
 
+  x86-openblocks-iot-vx2:
```

Comment:
This one has also been offline for a very long time:
https://lava.ciplatform.org/scheduler/device_type/x86-openblocks-iot-vx2